src/Wizard/Platform/Readline/ReadlinePlatform.ts

```diff
@@ -0,0 +1,161 @@
+import { Platform } from '../Platform';
+import { TestType } from '../../Models';
+import { Tokens } from '../../';
+import { ConnectivityAnalyzer } from '../../Services';
+import { inject, injectable } from 'tsyringe';
+import readline from 'readline';
+import { URL } from 'url';
+
+@injectable()
+export class ReadlinePlatform implements Platform {
+  private rl: readline.Interface;
+  private readonly delimiter = '\n\r--\n';
+
+  constructor(
+    @inject(Tokens) private readonly tokens: Tokens,
+    @inject(ConnectivityAnalyzer)
+    private readonly connectivityService: ConnectivityAnalyzer
+  ) {}
+
+  public async start(): Promise<ReadlinePlatform> {
+    this.rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout
+    });
+
+    this.rl.once('connected', async () => {
+      console.log('Welcome to the NexPloit Network Testing wizard!');
+      await this.configure();
+    });
+
+    this.rl.emit('connected');
```

Comment:
You can avoid such hacky code by omitting return value.

Reply:
What do you mean?